core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala

```diff
@@ -49,6 +50,26 @@ class JournalTableConfiguration(config: Config) {
   override def toString: String = s"JournalTableConfiguration($tableName,$schemaName,$columnNames)"
 }
 
+class JournalMetadataTableColumnNames(config: Config) {
+  private val cfg = config.asConfig("tables.journalMetadata.columnNames")
+  val id: String = cfg.as[String]("id", "id")
+  val persistenceId: String = cfg.as[String]("persistenceId", "persistence_id")
+  val maxSequenceNumber: String = cfg.as[String]("maxSequenceNumber", "max_sequence_number")
+  val maxOrdering: String = cfg.as[String]("maxOrdering", "max_ordering")
+  val minOrdering: String = cfg.as[String]("minOrdering", "min_ordering")
+
+  override def toString: String =
+    s"JournalMetadataTableColumnNames($id,$persistenceId,$maxSequenceNumber,$maxOrdering,$minOrdering)"
+}
+
+class JournalMetadataTableConfiguration(config: Config) {
+  private val cfg = config.asConfig("tables.journalMetadata")
+  val tableName: String = cfg.as[String]("tableName", "journal_metadata")
```

Comment:
@tiagomota typo on this line? `journal_metadata` is wrong no?

Reply:
No? Or am I missing something? 🤔 That is intended. The config will use `journal_metadata` as the default name of the metadata table, unless another is provided.